.github/workflows/e2e-tests.yml

```diff
@@ -186,9 +187,11 @@ jobs:
 
       - name: Run EE Cypress tests on ${{ matrix.folder }}
         if: matrix.edition == 'ee' && github.event_name != 'schedule'
+        # Run tests inside the folder, but also search for and run related tests
+        # that might live elsewhere. We tag such tests with `@foo` (e.g. @actions)
         run: |
           yarn run test-cypress-run \
-          --env grepTags="-@quarantine" \
+          --env grepTags="@${{ matrix.folder }}+-@quarantine" \
```

Comment:
Are we ok with a bit of redundancy until we switch to fully tagged tests?

In practice, this means that the runner which runs E2E tests in `actions` folder will also pick up tests from other folders that are tagged with `@actions` tag.
For example, there is one in the `admin` group:
https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/admin/databases/actions.cy.spec.js

This test will run in both `admin` and `actions` runners.

Reply:
@uladzimirdev "no" it is :)
It doesn't work anyway because all tests that ARE NOT tagged are simply skipped.
![image](https://github.com/metabase/metabase/assets/31325167/7c756938-5fdd-43d5-b390-a4d0c7a5c32c)

> What prevents us from moving away from folder based running?

There are no technical limitations.
We just need to invest some time in proper tagging, and the timing is never right for that.